src/main/java/uk/ac/ebi/eva/commons/mongodb/repositories/VariantRepositoryImpl.java

```diff
@@ -122,12 +122,21 @@ public Long countByRegionsAndComplexFilters(List<Region> regions, List<VariantRe
         query.with(pageable1);
 
         if (exclude != null && !exclude.isEmpty()) {
+            exclude = substituteExcludeFields(exclude);
```

Comment:
I'm not sure the change should be done at the library level. `sourceEntries` is an API string and in order to convert it we created [this utility class in eva-ws](https://github.com/EBIvariation/eva-ws/blob/develop/eva-server/src/main/java/uk/ac/ebi/eva/server/Utils.java).